tests/screenshot/diff_screenshots.js

```diff
@@ -31,35 +31,55 @@ var old_dir = 'tests/screenshot/outputs/old/';
 var new_dir = 'tests/screenshot/outputs/new/';
 var diff_dir = 'tests/screenshot/outputs/diff/';
 
-// TODOs:
-// - iterate over all test cases
-// - use the name of the test case file as the name of the test
-// - improve doneReading to not rely on globals like this
-// - write or find basic html page that lets you open pairs of outputs + diffs
+if (!fs.existsSync(diff_dir)){
+  fs.mkdirSync(diff_dir);
+}
+
+function getTestList() {
+  var file = fs.readFileSync('tests/screenshot/test_cases/test_cases.json');
```

Comment:
Should this path be up with the new dir?

Reply:
Done.